code/modules/clothing/under/miscellaneous.dm

```diff
@@ -809,9 +809,37 @@
 	worn_state = "kimono"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 
-/obj/item/clothing/under/itd
+/obj/item/clothing/under/itdofficial
 	desc = "It's a snazzy dark purple formal shirt worn by someone who works in the \"Internal Tribunal Department\", it has silver markings and a white nametag."
-	name = "Internal Tribunal uniform"
+	name = "Internal Tribunal Official uniform"
```

Comment:
we don't use the  word official in this organization in reference to a person who act as representative of their branch

Reply:
> we don't use the word official in this organization in reference to a person who act as representative of their branch

Meant to be for those who report to the site, like full on ITD Officials.